test/hydrated_test.dart

```diff
@@ -58,7 +56,7 @@ void main() {
     final second = SerializedClass(false, 42);
 
     /// null before hydrate
-    expect(subject.value, equals(null));
+    expect(subject.valueOrNull, equals(null));
```

Comment:
```suggestion
    expect(subject.valueOrNull, isNull);
```
Would it be a bit more readable? It would be great if you'll go through all tests and replace the following statements: 
- equals(null) -> isNull
- equals(true) -> isTrue

Reply:
Done!